src/main/java/com/questhelper/QuestHelperConfig.java

```diff
@@ -616,4 +690,16 @@ default boolean showCompletedQuests()
 	{
 		return false;
 	}
+
+	@ConfigItem(
```

Comment:
Open question: Should this filter stick around after a restart?

If it does, the name (& key name) should be changed to match its name in the main panel